pkg/controller/command/verifiable/command.go

```diff
@@ -842,7 +840,24 @@ func (o *Command) parsePresentationRequest(request *PresentationRequest,
 		return nil, nil, nil, fmt.Errorf("failed to prepare proof options: %w", err)
 	}
 
-	return vcs, presentation, opts, nil
+	return nil, presentation, opts, nil
+}
+
+func (o *Command) parsePresentationRequest(request *PresentationRequest,
+	didDoc *did.Doc) ([]interface{}, *verifiable.Presentation, *ProofOptions, error) {
+	if request.ProofOptions == nil || request.SignatureType == "" {
```

Comment:
why was the check for ProofOptions == nil added? I don’t see it in the prior code.

@soluchok @rolsonquadras @sudeshrshetty

Reply:
It was missing. `PresentationRequest` inherits `*ProofOptions`. Without this check, it will panic.